electricitymap/contrib/config/co2eq_parameters.py

```diff
@@ -0,0 +1,60 @@
+"""Contains a function to make co2eq parameter dicts from
```

Comment:
This could potentially put in the same file as the functions in zones.py (and reading.py) and could be called utils.py or something. Similar with config_types.py.